From 3203b78d1c4ff73557bc2ea88323ea4d65f55c36 Mon Sep 17 00:00:00 2001
From: kathiravan <kathiravan.s@ltts.com>
Date: Wed, 8 Apr 2020 14:29:03 +0530
Subject: [PATCH 6/9] Background color changed

---
 src/mainMenu/view.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mainMenu/view.js b/src/mainMenu/view.js
index a3fb675..252a96d 100644
--- a/src/mainMenu/view.js
+++ b/src/mainMenu/view.js
@@ -2,7 +2,7 @@ import { List } from './../core/list.js'
 import Config from './config.js'
 
 export function getBackground() {
-  return { x: 0, y: 0, rect: true, color: 0xffff0000, w: 1920, h: 1080 }
+  return { x: 0, y: 0, rect: true, color: 0xff000000, w: 1920, h: 1080 }
 }
 
 export function getMainList() {
-- 
2.23.0.windows.1

